waspc/packages/deploy/src/providers/railway/commands/setup/SetupCmdOptions.ts

```diff
@@ -7,4 +7,5 @@ import {
 export interface SetupCmdOptions extends CommonCmdOptions, SecretsOptions {
   existingProjectId: RailwayProjectId | null;
   workspace: string | null;
+  dbImage?: string;
```

Comment:
Why is this guy `?` and the rest is `| null`? Let's be uniform unless there is a strong reason? You can make this as the rest, of make the rest be as it is.

Reply:
The correct way is `?`, as the arguments parser will just not set that property if it is not passed. I will turn everything into `?`.